aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -617,7 +617,7 @@ Resources:
 <%=  component 'alarms'%>
 <% end -%>
 
-
+<%=  component 'browser_event_logging'%>
```

Comment:
Might be handy to rename this to `logging` and then Infrastructure team will add other logging Resources to this component in the future that are currently configured manually, provisioned in the top level template, or in some standalone template.

Reply:
Done!